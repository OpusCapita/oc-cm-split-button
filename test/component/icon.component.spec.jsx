/* eslint-disable no-unused-expressions */
import React from 'react';
import renderer from 'react-test-renderer';
import Icon from '../../src/icon.component';

describe('Icon', () => {
  test('renders correctly', () => {
    const icon = renderer
      .create(<Icon faIcon="fa fa-check" color="red" size="10rem" />)
      .toJSON();
    expect(icon).toMatchSnapshot();
  });
});
