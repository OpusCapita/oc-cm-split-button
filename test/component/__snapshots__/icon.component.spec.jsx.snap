// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon renders correctly 1`] = `
<i
  className="sc-bdVaJa fa fa-check haGCgV"
  color="red"
  size="10rem"
/>
`;
